### ISSUE TITLE
Open documentation needs updating

### ISSUE THREAD

**Issue by pedromachados** [astronomerio/astronomer#33]
This section of the documentation is hard to follow:

https://open.astronomer.io/airflow/index.html

I noticed this commit https://github.com/astronomerio/astronomer/commit/31f95f436fd9a11d5bffac5c968683c96002dca1#diff-364439c8141492fe1670af4f51c33125 removed the start/stop scripts.

What is the recommended way to start the services now? The commit message mentions the CLI but I could not find any documentation on using the CLI with Open.

I tried `docker-compose up` but it does not seem to be pointing to the onbuild image that reads requirements.txt and packages.txt so dependencies are not installed.

The document also mentions the .astro directory, which I don't believe is created by this version of the platform.

**Comment by schnie** [comment]
Hey @pedromachados, we'll be updating the docs soon. We are in the process of moving some of this functionality to our CLI, but that's under very active development at this point. 

For now, running something like `docker-compose --scale worker=2` or just plain `docker-compose up` is the way to go. 

To run a custom image, you can change to the directory of your project and just run something like `docker built -t mycompany/airflow:latest .`. If you've previously worked with the project in this directory it should have a `Dockerfile` with `FROM astronomerinc/ap-airflow:latest-onbuild`. If that file does not exist, you can simply create it.

Now that you have a base image, you can change the values inside the `.env` file next to the `docker-compose.yaml` file. You should be able to change `IMAGE_NAME`, to whatever you named it before (mycompany/airflow), and set `AIRFLOW_HOME` to path to your actual project. This will mount several directories from the image to the host so you can hack on your python code without restarting the docker-compose project. You will have to restart if you add anything to `packages.txt` or `requirements.txt` though.

This was basically what the old `start` script did under the hood.

Hope that helps for now. @tedmiston and @andscoop can you guys help out with these docs and/or brainstorming any other ideas here?

**Comment by ryw** [comment]
@tedmiston closing this out today? If not, remove from milestone

**Comment by tedmiston** [comment]
@ryw Yes, I'll close it today

**Comment by ryw** [comment]
@tedmiston ... only one other issue left, it's you vs. brad :)

**Comment by tedmiston** [comment]
Mostly working on Kafka today so he'll probably win that one.  This one is end of day for me.

**Comment by tedmiston** [comment]
Closed via https://github.com/astronomerio/astronomer/pull/39
